discovery-provider/src/queries/get_savers_for_playlist.py

```diff
@@ -25,30 +25,16 @@ def get_savers_for_playlist(args):
         if playlist_entry is None:
             raise exceptions.NotFoundError('Resource not found for provided playlist id')
 
-        # Subquery to get all (user_id, follower_count) entries from Follows table.
-        follower_count_subquery = (
-            session.query(
-                Follow.followee_user_id,
-                func.count(Follow.followee_user_id).label(response_name_constants.follower_count)
-            )
-            .filter(
-                Follow.is_current == True,
-                Follow.is_delete == False
-            )
-            .group_by(Follow.followee_user_id)
-            .subquery()
-        )
-
         # Get all Users that saved Playlist, ordered by follower_count desc & paginated.
         query = (
             session.query(
                 User,
                 # Replace null values from left outer join with 0 to ensure sort works correctly.
-                (func.coalesce(follower_count_subquery.c.follower_count, 0))
+                (func.coalesce(AggregateUser.follower_count, 0))
```

Comment:
Do we still have to coalesce these? Comment applies in a few places.

Reply:
I think so, because technically if the user was just created and we didn't refresh the mat view, then the user would not exist in the joined mat view and return null, I believe. I left this here purposely just in case. let me know if that makes sense.